adafruit_sdcard.py

```diff
@@ -81,6 +81,7 @@ class SDCard:
 
         :param ~busio.SPI spi: The SPI bus
         :param ~digitalio.DigitalInOut cs: The chip select connected to the card
+        :param int freq: The SPI data rate to use after card setup
```

Comment:
Please name this `frequency` or `baudrate`. Complete words are more understandable because it's easier to look up the meaning.

Reply:
Ok, going for baudrate to keep it in line with SPI name, even if it's strange from an sdcard perspective, where the term data_rate seems more common.